js/babel-plugin-sprockets-commoner-internal/index.js

```diff
@@ -25,7 +25,9 @@ if (typeof Object.assign != 'function') {
 
 var fs = require('fs');
 var dirname = require('path').dirname;
+var join = require('path').join;
 var resolve = require('browser-resolve').sync;
+var emptyModule = join(__dirname, 'node_modules', 'browser-resolve', 'empty.js');
```

Comment:
Why? It's what empty modules resolve to https://github.com/defunctzombie/node-browser-resolve/blob/a238dd1ca8db9319cdd6b6340358a09f298c9156/index.js#L56